addons/velero/1.6.2/install.sh

```diff
@@ -150,10 +162,15 @@ function velero_patch_http_proxy() {
 function velero_change_storageclass() {
     local src="$1"
     local dst="$2"
+    local disable_kustomization="$3"
 
     if kubectl get sc longhorn &> /dev/null && \
     [ "$(kubectl get sc longhorn -o jsonpath='{.metadata.annotations.storageclass\.kubernetes\.io/is-default-class}')" = "true" ]; then
         render_yaml_file "$src/tmpl-change-storageclass.yaml" > "$dst/change-storageclass.yaml"
-        insert_resources "$dst/kustomization.yaml" change-storageclass.yaml
+        if [ -z "$disable_kustomization" ]; then
+            insert_resources "$dst/kustomization.yaml" change-storageclass.yaml
```

Comment:
does this have to be $dst/change-storageclass.yaml

Reply:
Per our discussion, we don't need this since all the kustomize resources are in the same directory (and not nested).